app/assets/stylesheets/searchworks4/facets.css

```diff
@@ -25,3 +25,35 @@
     border-top: 1px solid var(--stanford-30-black);
   }
 }
+
+.facet-toggle-button{
+  &:not([aria-expanded="true"])::before {
+    font-family: 'bootstrap-icons';
+    content: "\F56B";
+    margin-right: 0.25em;
+    vertical-align: top;
+  }
+
+  &[aria-expanded="true"]{
+    border: none;
+    text-align: left;
+    &::after{
+      font-family: 'bootstrap-icons';
+      content: "\F62A";
+      float: right;
+      font-size: 1.25rem;
+    }
+  }
+}
+
+.mini-bento.btn {
+  &[aria-expanded="true"]{
+    display: none;
+  }
+}
+
+@media (width > 991.98px) {
+  #mini-bento.collapse:not(.show) {
```

Comment:
why not `.alternate-catalog`?  I feel like this is introducing a second way of addressing the same components.